bazel/framework_imports_extractor.bzl

```diff
@@ -36,7 +36,8 @@ def _framework_imports_extractor(ctx):
     ctx.actions.run_shell(
         inputs = [framework_zip],
         outputs = outputs,
-        command = "unzip -qq {} -d {}".format(framework_zip.path, ctx.bin_dir.path),
+        # Workaround for https://github.com/bazelbuild/rules_apple/issues/1489
+        command = "unzip -qq {} -d {} || true".format(framework_zip.path, ctx.bin_dir.path),
```

Comment:
If the unzip fails won't the required output files not be created

Reply:
In my experience the CRC failures cause the process to exit with a non-zero code, but the resulting files still match.